### PR TITLE
Handle ssl errors

### PR DESCRIPTION
**Pull request by dpopp07**
Adds a specific handler for ICP SSL errors. Also fixes the dreaded "Body is HTTP blah blah" error with a more descriptive one.

### CODE REVIEW

**Comment by mediumTaj** [comment]
dustin was this meant to go to `master`?

**Comment by dpopp07** [comment]
Nope, updated

File: lib/requestwrapper.ts [hunk 1]
```diff
@@ -270,14 +270,20 @@ export class RequestWrapper {
...
-      error.body = axiosError.request;
-
+      error.message = axiosError.message;
+      error.statusText = axiosError.code;
+      error.body = 'Response not received - no connection was made to the service.';
+
+      // when a request to a private cloud instance has an ssl problem, it never connects and follows this branch of the error handling
+      if (isSelfSignedCertificateError(axiosError)) {
+        error.message = `If you're trying to call a service on ICP or Cloud Pak for Data, you ` +
+          `may not have a valid SSL certificate. If you need to access the service without setting that up, try using ` +
```
Comment (germanattanasio) [inline lib/requestwrapper.ts]: What about?

> The service you are trying to call doesn't use valid certificates. You can disable the SSL certificate validation using the `disableSslVerification` option.

Comment (dpopp07) [reply lib/requestwrapper.ts]: I'm fine with that. I just copied the message that @lpatino10 has in Java for consistency. Would you rather this change?

**Comment by SirSpidey** [comment]
We usually state the error and the possible response. How about this?

The connection failed because the SSL certificate is not valid. To use a self-signed certificate, set the `disableSslVerification` parameter in`HttpConfigOptions`.

**Comment by dpopp07** [comment]
I'm fine with any of these. I just think that currently, all of the other SDKs are using the message in the PR, so do we want to change all of them? @mediumTaj

**Comment by mediumTaj** [comment]
Yes once we settle i will create an issue

**Pull request merged**